iceoryx_posh/test/moduletests/test_popo_publisher_port.cpp

```diff
@@ -63,10 +63,10 @@ class PublisherPort_test : public Test
     static constexpr size_t MEMORY_SIZE = 1024 * 1024;
     uint8_t m_memory[MEMORY_SIZE];
     static constexpr uint32_t NUM_CHUNKS_IN_POOL = 20;
-    static constexpr uint32_t SMALL_CHUNK = 128;
-    static constexpr uint32_t BIG_CHUNK = 256;
+    static constexpr uint64_t SMALL_CHUNK = 128;
+    static constexpr uint64_t BIG_CHUNK = 256;
 
-    static constexpr uint32_t USER_PAYLOAD_ALIGNMENT = iox::CHUNK_DEFAULT_USER_PAYLOAD_ALIGNMENT;
+    static constexpr uint64_t USER_PAYLOAD_ALIGNMENT = iox::CHUNK_DEFAULT_USER_PAYLOAD_ALIGNMENT;
```

Comment:
This also. If you changed it to prevent casts in the tests, then this can be left as it is now.